WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -660,6 +697,27 @@ class PostListViewModel @Inject constructor(
         }
     }
 
+    private fun loadPostFromRemote(localPostId: Int) {
+        // We need network connection to load a remote post
+        if (!checkNetworkConnection()) {
+            return
+        }
+
+        val post = postStore.getPostByLocalPostId(localPostId)
+        if (post != null) {
+            dispatcher.dispatch(PostActionBuilder.newFetchPostAction(RemotePostPayload(post, site)))
+        }
+        localPostIdForConflictResolutionDialog = null
+    }
+
+    private fun loadPostFromLocal(localPostId: Int) {
```

Comment:
Again, this might be a design question, but when the user selects to edit the local post, shouldn't we push the current post to remote so the conflict is resolved? cc @megsfulton

Reply:
technically that's right, but just to add to this (and maybe making it less complex), if they choose to edit the local copy, it'll be sent to the remote side as soon as they exit the editor, so in the end it's probably the same result.